docs/getting-started/quick-start.md

```diff
@@ -72,15 +67,107 @@ import { FormatPainter, SlashCommand } from 'ckeditor5-premium-features';
 
 import 'ckeditor5/dist/styles.css';
 
-ClassicEditor
+await ClassicEditor
 	.create( document.querySelector( '#editor' ), {
 		plugins: [ Essentials, FormatPainter, SlashCommand ],
 		toolbar: [ 'formatPainter' ],
 		licenseKey: 'your-license-key'
 	} )
-	.catch( err => {
-		console.error( err );
-	} );
+```
+
+## Using CKEditor&nbsp;5 from CDN
+
+CDN is an effortless method of running CKEditor 5. You can start using it in just a few steps and with a few tags.
+
+### Obtain a license key
+
+To activate CKEditor&nbsp;5 premium features, you will need a commercial license. If you wish to purchase a commercial CKEditor&nbsp;5 license or a license to one of the premium features, [contact us](https://ckeditor.com/contact/?sales=true#contact-form) to receive an offer tailored to your needs. To obtain an activation key, please follow the {@link support/license-key-and-activation License key and activation} giuide.
+
+<!-- Not sure we should duplicate this in QS: TODO: Describe the steps to obtain a license key, attaching some screenshots. -->
+
+### CDN setup
+
+Start by attaching a link to the style sheet. It contains all styles for the editor's UI and content. You can also include your styles if you like. Refer to the content styles guide for more information.
```

Comment:
Add link to the `content styles` guide.

Reply:
Sure, but we don't have this guide in the new structure yet.